api/src/db/seeds/development/settings.js

```diff
@@ -0,0 +1,15 @@
+const { cache } = require('../../../config')
+
+const settings = {
+  'osmesa-db': 'postgres://postgres@localhost:5434/postgres'
+}
+
+exports.seed = (knex) => knex('settings').del() // delete entries
+  .then(async () => {
+    const promises = Object.keys(settings).map(key => {
+      cache.put(key, settings[key])
```

Comment:
Consider adding an expiration to all the uses of `cache.put()`. There needs to be some way to expire and/or invalidate the cache. The package's docs say

> If time isn't passed in, it is stored forever

Reply:
If we are using the cache only to keep the settings in memory, wouldn't invalidating the values require some new logic to pull the settings again from the database? In what cases do we need to invalidate the cache?

Reply:
@kamicut yes I see what you mean. The only use case I can think of is: what if the admin wanted to update a setting with a SQL update instead of using the UI admin interface? The in database and the in-memory settings would stay different until restart. If that is fine, then ignore this one.

Reply:
That makes sense, I think we want to make sure that "if you are hitting the database you know what you're doing" and that's not available to app admins but rather devops. In the case of devops I would lean on the side of "if you're touching the database you are probably restarting the service"